core/io/image.cpp

```diff
@@ -2594,6 +2594,11 @@ Error Image::compress(CompressMode p_mode, CompressSource p_source, float p_loss
 }
 
 Error Image::compress_from_channels(CompressMode p_mode, UsedChannels p_channels, float p_lossy_quality) {
+	if (data.is_empty()) {
+		ERR_PRINT_ONCE_ED("Empty Image data.");
+		return ERR_INVALID_DATA;
+	}
```

Comment:
Why print once? I'd go for:
```suggestion
	ERR_FAIL_COND_V(data.is_empty(), ERR_INVALID_DATA);
```
instead (not sure if a dedicated error message is needed, I think `data.is_empty()` condition is quite clear).

Reply:
The error is spamming a lot inside `_process` (see original issue #70197) 
Although it doesn't make sense to compress image every frame.
![image](https://user-images.githubusercontent.com/32661049/208349167-9d592fb8-42dd-4629-b30d-c20ea5074b48.png)